cmds/deploy.go

```diff
@@ -217,6 +223,7 @@ func NewCmdDeploy(f *cmdutil.Factory) *cobra.Command {
 		},
 	}
 	cmd.PersistentFlags().StringP("domain", "d", defaultDomain(), "The domain name to append to the service name to access web applications")
+	cmd.PersistentFlags().StringP("apidomain", "", "--domain", "if set the fabric8 pod uses that domain to call the console")
```

Comment:
Maybe we call the param `oauth-authorize-uri` just to be clear? The reason it was domain before was because we reused that as a suffix for the console route as well as the authorize URI - as you've said this isn't really valid & needs to be 2 separate params.